supabase-cli installation via pnpm is broken

**Describe the bug**

supabase-cli installation via pnpm is broken


**To Reproduce**
Follow the [quick start docs](https://supabase.com/docs/guides/local-development?queryGroups=package-manager&package-manager=pnpm#quickstart) using pnpm

1. pnpm add supabase --save-dev
```
WARN  Failed to create bin at /node_modules/.bin/supabase. ENOENT: no such file or directory, open '/node_modules/.pnpm/supabase@2.22.6/node_modules/supabase/bin/supabase'
```

2. pnpm supabase --version
```
 ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "supabase" not found
```
 
**Expected behavior**

``pnpm supabase` should work

**Screenshots**
I

![Image](https://github.com/user-attachments/assets/e40b545c-7935-434c-8aff-dc2de5435a4a)

**System information**
Rerun the failing command with `--create-ticket` flag.
 - Ticket ID: <supabase command not work>
 - Version of OS: Fedora Linux 42
 - Version of CLI: 2.22.6
 - Version of Docker: Podman 5.4.2
 - Versions of services: <supabase command not work>

**Additional context**
If applicable, add any other context about the problem here.
 - Browser Chrom
 - Version of supabase-js  v2.22.6
 - Version of Node.js 23.11.0

--- a/apps/docs/content/guides/local-development.mdx
+++ b/apps/docs/content/guides/local-development.mdx
@@ -24,7 +24,7 @@ Develop locally while running the Supabase stack on your machine.
     </TabPanel><TabPanel id="pnpm" label="pnpm">
 
     ```sh
-    pnpm add supabase --save-dev
+    pnpm add supabase --save-dev --allow-build=supabase
     ```
 
     </TabPanel><TabPanel id="brew" label="brew">

